pdns/ws-auth.cc

```diff
@@ -928,14 +929,27 @@ static bool isValidMetadataKind(const string& kind, bool readonly) {
   return found;
 }
 
+// this is easier as macro since we need UeberBackend instance in most places
```

Comment:
Not terribly happy with this macro, but I do not see a way around it..

Reply:
yes, I had the same feeling